subhub/customer.py

```diff
@@ -134,3 +137,17 @@ def has_existing_plan(customer: Customer, plan_id: str) -> bool:
             ]:
                 return True
     return False
+
+
+def __validate_origin_system(origin_system: str):
+    """
+    This function validates a request's origin_system to validate that is permitted.  This
+    allows us to ensure that callers are permitted by configuration to interact with this application.
+    :param origin_system: The originating system in Mozilla
+    """
+    if origin_system not in CFG.ALLOWED_ORIGIN_SYSTEMS:
+        csv_origin_systems = ",".join(map(str, CFG.ALLOWED_ORIGIN_SYSTEMS))
```

Comment:
This seems unnecessary.  Doesn't the ALLOWED_ORIGIN_SYSTEMS property already return a list?  The __str__ or __repr__ version of the a list of strings isn't horrible to look at.
```
In [3]: import os

In [4]: os.environ["ALLOWED_ORIGIN_SYSTEMS"] = "Firefox,Salesforce"

In [5]: class Foo():
   ...:     @property
   ...:     def ALLOWED_ORIGIN_SYSTEMS(self):
   ...:         return os.environ["ALLOWED_ORIGIN_SYSTEMS"].split(',')
   ...: 

In [6]: foo = Foo()

In [7]: foo.ALLOWED_ORIGIN_SYSTEMS
Out[7]: ['Firefox', 'Salesforce']
```

Reply:
if not:
```
In [9]: ','.join(foo.ALLOWED_ORIGIN_SYSTEMS)
Out[9]: 'Firefox,Salesforce'
```

Reply:
Yup, I’ll update it.

Reply:
Yup, I just didn’t inline all of my changes after the lower() removal.  On it and sorry about that!